packages/nimble-components/src/icon-base/tests/icons.spec.ts

```diff
@@ -16,20 +16,14 @@ describe('Icons', () => {
             const svg = template.content.querySelector('svg');
             return svg!;
         };
-        const getPaths = (svg: SVGElement): SVGPathElement[] => Array.from(svg.querySelectorAll('path'));
 
         parameterizeSpec(nimbleIcons, (spec, name, value) => {
             spec(`for icon ${name}`, () => {
                 const svg = getSVGElement(value.data);
-                const paths = getPaths(svg);
                 expect(svg).toBeTruthy();
                 expect(svg.getAttribute('viewBox')).toBeTruthy();
                 expect(svg.getAttribute('height')).toBeNull();
                 expect(svg.getAttribute('width')).toBeNull();
-                expect(svg.querySelector('defs')).toBeNull();
-                for (const path of paths) {
-                    expect(path.getAttribute('style')).toBeNull();
-                }
```

Comment:
Do we think these are not generally useful? This is a parameterized spec, we could choose specific exceptions instead of disabling for all icons. i.e. one test with the more thorough tests and one with a smaller subset.

Reply:
pushed a change that preserves the style exclusion tests for other icons